guides/common/assembly_building_cloud-images.adoc

```diff
@@ -2,10 +2,6 @@ include::modules/con_building_cloud_images.adoc[]
 
 include::modules/proc_creating-custom-rhel-images.adoc[leveloffset=+1]
 
-include::modules/proc_creating-a-rhel7-image.adoc[leveloffset=+1]
```

Comment:
I think these are all client bits and should stay

Reply:
Addressed :+1: